WordPress/Classes/System/WPGUIConstants.m

```diff
@@ -1,11 +1,15 @@
 #import "WPGUIConstants.h"
 
-const CGFloat WPAlphaFull = 1.0f;
-const CGFloat WPAlphaZero = 0.0f;
+const CGFloat WPAlphaFull                   = 1.0f;
+const CGFloat WPAlphaZero                   = 0.0f;
 
-const CGFloat WPColorFull = 1.0f;
-const CGFloat WPColorZero = 0.0f;
+const CGFloat WPColorFull                   = 1.0f;
+const CGFloat WPColorZero                   = 0.0f;
 
-const CGFloat WPAnimationDurationDefault = 0.33f;
-const CGFloat WPAnimationDurationFast = 0.15f;
-const CGFloat WPAnimationDurationFaster = 0.07f;
\ No newline at end of file
+const CGFloat WPAnimationDurationDefault    = 0.33f;
+const CGFloat WPAnimationDurationFast       = 0.15f;
+const CGFloat WPAnimationDurationFaster     = 0.07f;
+
+const CGFloat WPTableViewTopMargin          = 40.0f;
+const CGRect WPTableHeaderPadFrame          = {0.0f, 0.0f, 0.0f, 40.0f};
+const CGRect WPTableFooterPadFrame          = {0.0f, 0.0f, 0.0f, 48.0f};
```

Comment:
I'm digging it :)
Part of me wonders if this would fit better in a WPStyleGuide category. I think it would be nice to keep anything related to UI style as a part of that class. 
The really really nitpicky part of me says we could leave off the `f` and let the compiler worry about exactly what flavor of CGFloat the constants should be.